libs/pika/thread_pools/include/pika/thread_pools/scheduling_loop.hpp

```diff
@@ -49,7 +49,7 @@ namespace pika::threads::detail {
                            "scheduler({}), worker_thread({}), thread({}), "
                            "description({}), old state({}), new state({})",
             *scheduler.get_parent_pool(), scheduler, num_thread,
-            get_thread_id_data(thrd),
+            static_cast<void*>(get_thread_id_data(thrd)),
```

Comment:
Yes, or even better `get_thread_id_data(thrd)->get_thread_id()` as it is in some other calls in the same file. I'll change it to that.

Reply:
Turns out that didn't quite work out. In some cases the `thrd` is already terminated in which case the internal coroutine pointer has already been reset and `get_thread_id()` triggers an assertion.

Luckily there's an even simpler solution: simply print `thrd` directly since it has a `formatter` specialization...